.jenkins/env.sh

```diff
@@ -0,0 +1,15 @@
+envdir="${envloc}/env"
```

Comment:
`pushd` and `popd` are bash builtins, so this file needs `#!/usr/bin/env bash` to be proper.

Reply:
Otherwise this seems like the `.sh` or sh shell could execute it, which it cannot.

Reply:
🚀